### PR TITLE
engine: 25.0.2 release notes

### PR DESCRIPTION
**Pull request by vvoland**
### Proposed changes

Adds release notes for moby v25.0.2

(Full credit goes to @dvdksn - I just filled the PR links)

### CODE REVIEW

File: content/engine/release-notes/25.0.md [hunk 1]
```diff
@@ -19,6 +19,54 @@ For more information about:
...
+affecting Docker Engine and its components.
+
+| CVE                                                         | Component     | Fix version | Severity         |
+| ----------------------------------------------------------- | ------------- | ----------- | ---------------- |
+| [CVE-2024-21626](https://scout.docker.com/v/CVE-2024-21626) | runc          | 1.1.12      | High, CVSS 8.6   |
+| [CVE-2024-23651](https://scout.docker.com/v/CVE-2024-23651) | BuildKit      | 1.12.5      | High, CVSS 8.7   |
+| [CVE-2024-23652](https://scout.docker.com/v/CVE-2024-23652) | BuildKit      | 1.12.5      | High, CVSS 8.7   |
+| [CVE-2024-23653](https://scout.docker.com/v/CVE-2024-23653) | BuildKit      | 1.12.5      | High, CVSS 7.7   |
+| [CVE-2024-23650](https://scout.docker.com/v/CVE-2024-23650) | BuildKit      | 1.12.5      | Medium, CVSS 5.5 |
+| [CVE-2024-24557](https://scout.docker.com/v/CVE-2024-24557) | Docker Engine | 1.12.5      | Medium, CVSS 6.9 |
```
Comment (dvdksn) [inline content/engine/release-notes/25.0.md:43]: Oops, this version needs fixing. My oversight, sorry!

Comment (vvoland) [reply content/engine/release-notes/25.0.md]: Fixed, thanks! I looked at these and somehow still missed it 🙈

File: content/engine/release-notes/25.0.md [hunk 2]
```diff
@@ -19,6 +19,54 @@ For more information about:
...
+- [CVE-2024-21626](https://github.com/opencontainers/runc/security/advisories/GHSA-xr7r-f8xq-vfvv)
+- [CVE-2024-23651](https://github.com/moby/buildkit/security/advisories/GHSA-m3r6-h7wv-7xxv)
+- [CVE-2024-23652](https://github.com/moby/buildkit/security/advisories/GHSA-4v98-7qmw-rqr8)
+- [CVE-2024-23653](https://github.com/moby/buildkit/security/advisories/GHSA-wr6v-9f75-vh2g)
+- [CVE-2024-23650](https://github.com/moby/buildkit/security/advisories/GHSA-9p26-698r-w4hx)
+- [CVE-2024-24557](https://github.com/moby/moby/security/advisories/GHSA-xw73-rw38-6vjc)
+
+### Packaging updates
+
+- Upgrade containerd to [v1.6.28](https://github.com/containerd/containerd/releases/tag/v1.6.28).
```
Comment (sam-thibault) [inline content/engine/release-notes/25.0.md:64]: Wasn't this only for Moby 23?

Comment (vvoland) [reply content/engine/release-notes/25.0.md:64]: 1.7.13 is used only in CI/static packages. For the runtime usage we still package 1.6.

cc @thaJeztah correct me if I'm wrong please.

Comment (sam-thibault) [reply content/engine/release-notes/25.0.md:64]: I came across this note on the wiki checklist:

> moby/moby vendor: github.com/containerd/containerd v1.7.13 / v1.6.28 (TBD: containerd 1.6 has updated OTEL versions, and other changes not directly related to this CVE, so we can keep this for v25.0.3)

Comment (thaJeztah) [reply content/engine/release-notes/25.0.md:64]: We didn't update containerd vendoring, but;

- release packages (deb / rpm) for containerd 1.6.28 (includes runc 1.1.12)
- static packages contain containerd 1.7.13 (and runc 1.1.12)

**Review by sam-thibault — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**